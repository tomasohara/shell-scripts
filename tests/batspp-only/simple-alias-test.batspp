# Simple alias test

$ alias count-words='wc -w'

$ echo abc def ght | count-words
3
