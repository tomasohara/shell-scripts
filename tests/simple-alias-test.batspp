# Simple alias test

# Global Setup
$ alias count-words='wc -w'

$ echo abc def ght | count-words
3
