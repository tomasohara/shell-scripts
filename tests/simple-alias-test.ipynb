{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Simple alias test"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "alias count-words='wc -w'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n"
     ]
    }
   ],
   "source": [
    "echo abc def ght | count-words"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash",
   "language": "bash",
   "name": "bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
