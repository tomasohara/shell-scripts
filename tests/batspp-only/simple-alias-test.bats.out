1..2
ok 1 test-1
not ok 2 test-3
# (from function `test-3-actual' in file batspp-only/simple-alias-test.bats, line 66,
#  in test file batspp-only/simple-alias-test.bats, line 53)
#   `test-3-actual ' failed with status 127
# ========== echo abc def ght | count-words
#  ==========
# /home/aveey/tom-project/shell-scripts/tests/batspp-only/simple-alias-test.bats: line 66: count-words: command not found
